xbmc/win32/WIN32Util.cpp

```diff
@@ -216,18 +216,15 @@ bool CWIN32Util::PowerManagement(PowerState State)
       return false;
   }
 
-  // process OnSleep() events. This is called in main thread.
-  g_powerManager.ProcessEvents();
```

Comment:
I think @mkortstiege is correct that we still need this, it doesn't process anything but it calls the callbacks on g_powerManager to notify about suspend/resume events